src/components/Verse/VerseActionRepeatAudio.tsx

```diff
@@ -35,7 +40,7 @@ const VerseActionRepeatAudio = ({ verseKey, isTranslationView }: VerseActionRepe
         selectedVerseKey={verseKey}
         chapterId={chapterId.toString()}
         isOpen={isModalOpen}
-        onClose={() => setIsModalOpen(false)}
+        onClose={() => onActionTriggered()}
```

Comment:
This can be `onClose={onActionTriggered}` directly.

Reply:
you're right there's no need to wrap `onActionTriggered` in an anonymous function.
i'll modify it and push the changes.